WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowersUseCase.kt

```diff
@@ -43,8 +43,7 @@ import javax.inject.Named
 private const val BLOCK_ITEM_COUNT = 6
 private const val VIEW_ALL_PAGE_SIZE = 10
 
-class FollowersUseCase
-@Inject constructor(
+class FollowersUseCase(
```

Comment:
NitPick: We might want to consider making the constructor private so no-one can instantiate the class without the factory. Wdyt?

```class FollowersUseCase private constructor(```

Reply:
If it's private, we cannot access it from tests. I think it's fine if it's public

Reply:
We could use the factory in the tests, right? Unless you think that it wouldn't be a unit test anymore as we'd be using another class. But it's such a nitpick that it doesn't really matter.